docs/quickstart/basics.rst

```diff
@@ -8,46 +8,64 @@ Let's take a look at a very basic API using Flask-Rebar:
 
 .. code-block:: python
 
+   # todos.py
+
    from flask import Flask
    from flask_rebar import Rebar
-   from flask_rebar import ResponseSchema
-   from marshmallow import fields
+   from marshmallow import Schema, fields
 
 
    rebar = Rebar()
    registry = rebar.create_handler_registry()
 
 
-   class TodoSchema(ResponseSchema):
-       id = fields.Integer()
+   class TodoSchema(Schema):
+       # The task we have to do.
+       description = fields.String(required=True)
+
+       # The unique id for this Todo. Determined by whatever database we're using.
+       id = fields.Integer(
+           # We'll explain the dump_only=True shortly.
+           dump_only=True
+       )
 
 
    @registry.handles(
       rule='/todos/<id>',
       method='GET',
-      response_body_schema=TodoSchema()  # for versions <= 1.7.0, use marshal_schema
+      # Marshal bodies of 200 (default) responses using the TodoSchema.
+      response_body_schema=TodoSchema()
    )
    def get_todo(id):
-       ...
-       return {'id': id}
+       todo = _get_todo_or_404(id)  # Some helper function that queries our database.
```

Comment:
Shout out to https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.BaseQuery.first_or_404